### PR TITLE
fix(installation): remove old configs if `KeyError` | macOS installation

### LINKED ISSUE #274: Error while trying to open GUI in the latest PlantSeg version
Hello,
Facing an issue while trying to open the GUI for the latest version of PlantSeg 1.8.1 after an install in a fresh environment
`ModuleNotFoundError: No module named 'bioimage.io'`
(same error even on trying to open Napari GUI)

Even on installing bioimage.core separately
displays an error in the legacy_gui/gui_widgets.py
`KeyError: 'is_segmentation'`

### PR DESCRIPTION
**Pull request by qin-yu**
Fix #274.

Try to improve error message for users to deal with `KeyError` introduced by upgrade.

Maybe we can also just delete the folders for them? @lorenzocerrone

### CODE REVIEW

**Comment by qin-yu** [comment]
I've updated the installation documentation to address the `KeyError`, and I've also separated the macOS installation guide to potentially fix #273. Although I don't have access to an M-chip Mac for testing tonight, I followed the instructions from this PyTorch page and will verify it later. Feel free to test it if you have time.

image

**Comment by lorenzocerrone** [comment]
Check #276 before merging the new docs. The python version needs to be pinned on Mac.

File: docs/chapters/getting_started/installation.md [hunk 1]
```diff
@@ -29,30 +29,42 @@ The easiest way to install PlantSeg is by using the [conda (Anaconda)](https://w
...
+        ```bash
+        mamba create -n plant-seg -c pytorch -c nvidia -c conda-forge pytorch cpuonly pyqt plant-seg --no-channel-priority
+        ```
+
+=== "macOS"
+
+    * Apple Silicon version
+
+        ```bash
+        mamba create -n plant-seg -c pytorch -c conda-forge pytorch::pytorch pyqt plant-seg --no-channel-priority
```
Comment (lorenzocerrone) [inline docs/chapters/getting_started/installation.md:64]: python=3.11 #276

File: plantseg/run_plantseg.py [hunk 2]
```diff
@@ -21,8 +21,21 @@ def create_parser():
 def launch_gui():
     """Launch the GUI configurator."""
     from plantseg.legacy_gui.plantsegapp import PlantSegApp
-
-    PlantSegApp()
+    from plantseg import PATH_CONFIGS
+
+    try:
+        PlantSegApp()
+    except KeyError:
```
Comment (lorenzocerrone) [inline plantseg/run_plantseg.py:28]: looks good

**Review by lorenzocerrone — CHANGES_REQUESTED** [review]
please @qin-yu pin the python version in the docs. Besides that looks great, you can merge!

**Pull request merged**